torch/_prims/nvfuser_executor.py

```diff
@@ -171,6 +171,15 @@ def maybe_partition_graph(gm: GraphModule):
                 category=RuntimeWarning,
             )
         partitioned_graph = partitioner.fuse_partitions(partitions)
+
+        # Overwriting graph's __call__() function with nvfuser_execute
+        # This avoids the need to call the interpreter on the graph
+        for node in partitioned_graph.graph.nodes:
+            # TODO: use a better way to identify fused submodule
+            if node.op == "call_module" and "fused_" in node.name:
+                fused_module = getattr(partitioned_graph, node.name)
+                fused_module._wrapped_call = nvfuser_execute
```

Comment:
Don't do this. https://github.com/pytorch/torchdynamo/blob/042c574479c58cf68a5c100f0e08e7a86eb87fb3/torchdynamo/optimizations/training.py#L268-L321 shows the correct way to put in an opaque module that calls into the executor, and https://github.com/pytorch/torchdynamo/blob/042c574479c58cf68a5c100f0e08e7a86eb87fb3/torchdynamo/optimizations/training.py#L362-L371 shows how to do the graph replacement.

Reply:
Thanks! That's cleaner.